cumulusci/cli/tests/test_cci.py

```diff
@@ -1671,14 +1671,14 @@ def test_org_scratch_delete_not_scratch(self):
         with self.assertRaises(click.UsageError):
             run_click_command(cci.org_scratch_delete, runtime=runtime, org_name="test")
 
-    def test_org_scratch_delete_error(self):
+    @mock.patch("click.echo")
+    def test_org_scratch_delete_error(self, echo):
         org_config = mock.Mock()
         org_config.delete_org.side_effect = ScratchOrgException
         runtime = mock.Mock()
         runtime.keychain.get_org.return_value = org_config
-
-        with self.assertRaises(ScratchOrgException):
-            run_click_command(cci.org_scratch_delete, runtime=runtime, org_name="test")
+        run_click_command(cci.org_scratch_delete, runtime=runtime, org_name="test")
+        assert "org remove" in str(echo.mock_calls)
```

Comment:
This is fine, but I think it's also possible to use this pytest fixture to capture output: https://docs.pytest.org/en/stable/capture.html#accessing-captured-output-from-a-test-function